otherlibs/stdune-unstable/unix_error.ml

```diff
@@ -0,0 +1,279 @@
+type t = Unix.error
+
+(* CR-someday amokhov: It would be nice to derive this instead. For now let's
+   trust I haven't messed this up. *)
+let equal (x : t) (y : t) =
```

Comment:
I don't think you messed it up, but you could have also used polymorphic comparison.

Reply:
Yeah, I thought about it but I didn't want to put on my hazmat suit to touch polymorphic comparison.

My second option was to just use `fun _ _ -> false` since we probably don't care about the cutoff for errors when reading directories. Maybe I should have gone with this option.